java_tests/src/main/java/org/infinispan/client/hotrod/configuration/ServerConfigurationBuilder.java

```diff
@@ -8,11 +8,10 @@
  * @author Tristan Tarrant
  * @since 5.3
  */
-public class ServerConfigurationBuilder extends AbstractConfigurationChildBuilder implements Builder<ServerConfiguration> {
+public class ServerConfigurationBuilder {
```

Comment:
Vittorio, correct me if I'm wrong but removing the extension of AbstractConfigurationChildBuilder means that users won't be able to chain configuration elements in their code. E.g. when then write .AddServer().Host(..).Port(..), they won't be able to add .Ssl() anymore.